lib/vanagon/platform/windows.rb

```diff
@@ -174,21 +189,16 @@ def generate_msi_package(project)
           "mkdir -p $(tempdir)/{SourceDir,wix/wixobj}",
           "#{@copy} -r wix/* $(tempdir)/wix/",
           "gunzip -c #{project.name}-#{project.version}.tar.gz | '#{@tar}' -C '$(tempdir)/SourceDir' --strip-components 1 -xf -",
-          # Run the Heat command in a single pass
-          # Heat command documentation at: http://wixtoolset.org/documentation/manual/v3/overview/heat.html
-          #   dir <directory> - Traverse directory to find all sub-files and directories.
-          #   -ke             - Keep Empty directories
-          #   -cg             - Component Group Name
-          #   -gg             - Generate GUIDS now
-          #   -srd            - Suppress root element generation, we want to reference the default root element
-          #                     TARGETDIR in the project.wxs file, not a newly generated one.
-          #   -sreg           - Suppress registry harvesting.
-          "cd $(tempdir); \"$$WIX/bin/heat.exe\" dir SourceDir -v -ke -indent 2 -cg #{cg_name} -gg -srd -t wix/filter.xslt -sreg -out wix/#{project.name}-harvest.wxs",
+          "cd $(tempdir); \"$$WIX/bin/heat.exe\" dir #{app_source_path} #{app_heat_flags} -out wix/#{project.name}-harvest-app.wxs",
+          "cd $(tempdir); \"$$WIX/bin/heat.exe\" dir #{appdata_source_path} #{appdata_heat_flags}  -out wix/#{project.name}-harvest-programdata.wxs",
```

Comment:
@mckern it's actually not possible for CommonAppDataFolder to not exist, it's a windows default.